she_codes_news/news/forms.py

```diff
@@ -0,0 +1,39 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.forms import ModelForm
+from .models import NewsStory, Comment
+
+User = get_user_model()
+
+class StoryForm(ModelForm):
+    class Meta:
+        model = NewsStory
+        fields = ['title', 'pub_date', 'content', 'image_url', 'categories']
+        widgets = {
+            'pub_date': forms.DateInput(format=('%m/%d/%Y'),
+            attrs={
+                'class':'form-control',
+                'placeholder':'Select a date',
+                'type':'date'
+                }
+            ),
+        }
+
+ORDER_CHOICE= (
+    ('', "newest first"),
+    ('oldfirst', "oldest first")
+)
+
+class FilterForm(forms.Form):
+    categories = forms.ChoiceField(label="categories", choices=[('', '----'), ('clickbait','Clickbait'), ('politics','Politics'), ('travel','Travel'), ('badbitch', 'Bad bitch, its a genre')], required=False)
```

Comment:
really shouldn't hard code here at all. 
if you want to use hard-coded values, you should be putting these as a constant in your models.py and import it into here.